src/pe_2.py

```diff
@@ -0,0 +1,21 @@
+'''Even Fibonacci numbers
+Problem 2 
+Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will be:
+
+1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...
+
+By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.'''
+
+a,b=[],[]
+c,d=0,1
+
+while c<4000000:
+  a.append(c)
+  c,d=d,c+d
+  
+for num in a[1:len(a)]:
+  if num % 2 == 0:
+    b.append(num)
+    continue
```

Comment:
The `continue` keyword tells Python to skip the rest of what it would do in a loop, and move to the next item.  In this case, there's nothing more anyway, so `continue` actually doesn't do anything here!